.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
+}
```

Comment:
Delete, you should probably add `.vscode` to `$HOME/.gitignore_global`

Reply:
+1